src/main/java/fi/fmi/avi/model/swx/amd82/immutable/SpaceWeatherRegionImpl.java

```diff
@@ -41,9 +41,6 @@ public static Optional<SpaceWeatherRegionImpl> immutableCopyOf(final Optional<Sp
     public abstract Builder toBuilder();
 
     public static class Builder extends SpaceWeatherRegionImpl_Builder {
-        @Deprecated
-        Builder() {
-        }
 
         public static Builder from(final SpaceWeatherRegion value) {
```

Comment:
I think the `@Deprecated` annotation can be removed, but the empty constructor should be in place to retain the package-private visibility.